app/views/groups/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<h1 class="text-center mt-4">Transactions</h1>
+<div class="container">
+  <h4 class='text-center'>Total Amount :- <%= @group.entities.sum(:amount)%>$</h4>
+  <% @group.entities.order("created_at DESC").each do |entity| %>
+  <div class="row justify-content-md-center">
+    <div>
+      <div class="bg-white text-color card text-center shadow-lg mb-5 rounded">
+        <div class="card-body">
+          <div>
+            <h5 class="card-title text-color"><%= entity.name.capitalize%></h5>
+          </div>
+          <div class="card-text text-color">Amount: <%= entity.amount%>$</div>
+        </div>
+        <div class="card-footer text-muted">
+          <p> created <%= time_ago_in_words(entity.created_at) %> ago</p>
+        </div>
+      </div>
+    </div>
+  </div>
```

Comment:
- [ ] Based on the requirements, please make sure when the user clicks on the "Back" button (<), the user navigates to the home page, there is currently no back button on your page please add it 👍🏿

- [ ] Based on the requirements, when the user click (or taps) the "Save" button to create the new category, and it's taken to the transactions page for that category
- I currently can not save a transaction, see video below ⬇️ 

https://user-images.githubusercontent.com/69638013/202265981-c31589e5-22c6-4c96-9dcd-8afd43cd162b.mp4

Reply:
done